src/ui/Views/Shared/_Layout.cshtml

```diff
@@ -41,7 +41,7 @@
                 <ul id="proposition-links">
                     @if (User.Identity.IsAuthenticated)
                     {
-                    <li><a asp-controller="Auth" asp-action="Logout">Sign out</a> ( <a href="@profileUrl"> @name </a> ) </li>
+                    <li><a asp-controller="Auth" asp-action="Logout">Sign out</a> (<a href="@profileUrl"> @name </a>) </li>
```

Comment:
So close: Please also remove the spaces around the `@name`

Reply:
![image](https://user-images.githubusercontent.com/470137/42441131-74022d80-835f-11e8-8e87-ac13f78645ed.png)


I love those laser eye, if you ever want to give me a pair of souvenirs, can I have them in skewers.